README.md

```diff
@@ -1,52 +1,167 @@
 
-theoretical definition of API
+> In computer programming, an application programming interface (API) is a set of routines, protocols, and tools for building software and applications. - Wikipedia
+
+In its simplest form, an API in relation to Object Oriented programming is a class and the list of methods we define.  When creating a class, we are defining a guidebook on how to interact with other parts of the code.  We get to decide which methods and variables are public or private, essentially controlling how to interact with the class.  When we apply this concept to the web, we get web APIs like the GitHub API and Twitter API. From our Postman experiment, we saw how GitHub provides a way for us interact with the data on their system. Just like how a class provides a set of public methods to interact with, web APIs provide us with urls. The list of urls that GitHub provides on https://developer.github.com/v3 act as the public methods into their system.  The developers that created the API control which resources they want to share and who has access to them. 
 
 ## Ajax
 
-describe what ajax is and what it's used for
+*** SOMETHING HERE to make it flow better ***
+
+Asynchrounous Javascript and XML (Ajax) is a technique that is used in web applications.  It provides a way for content to be retrieved from a server and displayed without having to refresh the entire page.  Remember in your **blank application** where you had to create a list of **objects**?  Each time we filled out the form and clicked on **create**, the web browser would have to refresh the entire page to see the results sent back from our server.
+
+Single page applications provide a better user experience by allowing users to manipulate data on the server and see the results without having to refresh the page.  This is the power of Ajax in action.  In the background, requests are made to a web API using JavaScript.  As developers we can then choose to alter the displayed HTML based on the responses from the web API.
+
+Let's try an example. To start, create a folder on your computer called ```example```. Inside that folder create another folder called ```html```. Now create a file named ```index.html```, ```script.js``` and ```html/sentence.html```. Finally add the following to each file.
+
+#### index.html
+```html
+<!doctype html>
+<head>
+  <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.3/jquery.min.js"></script>
+    <script src="script.js"></script>
+</head>
+<body>
+  <p id="sentences">
+    Loading our page...
+  </p>
+</body>
+```
+
+#### html/sentence.html
+```html
+<p>Ajax is a brand of cleaning products, introduced by Colgate-Palmolive in 1947 for a powdered household and industrial cleaner.</p>
+<p>It was one of the company's first major brands.</p>
+```
+
+#### javascript.js
+```javascript
+// We should wait for the page to load before running our ajax request
+$(document).ready(function(){
```

Comment:
I’m a little wary of using jQuery for something like this. I can see arguments on both sides, but a simple XHR request could demonstrate all of the power of AJAX without the weight of jQuery. (It would be so cool to see a `fetch` polyfill used instead, too, but that might be advanced in its own way: requiring Promises, etc.)